README.md

```diff
@@ -1,56 +1,34 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Hello Microverse Project
 
+> This is a html webpage that contains a header with a h1 in it
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- HTML
+- CSS
 
 
-## Getting Started
+## Live Demo
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+[Live Demo Link](https://tekyrey.github.io/Hello-Microverse/index.html)
 
 
-To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
+- VS CODE
 
 ### Setup
-
-### Install
-
-### Usage
-
-### Run tests
-
-### Deployment
-
-
+Clone the repository using the command "git clone https://github.com/TekyRey/Hello-Microverse.git" in your local machine
+After cloning the repository, move into the folder you cloned into and open the index.html file using your browser
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
+- GitHub: [@githubhandle](https://github.com/TekyRey)
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```

Comment:
[OPTIONAL] I recommend you fix this section of your README by adding your name to the `Author1` tag as well as adding your Twitter handle (if you have one) and your LinkedIn profile (which you should definitely have). Moving forward, this will give you more visibility and will allow people interested in your projects to get in touch with you easier.